deploy/kubernetes/helm-charts/altinn-repositories/values.yaml

```diff
@@ -64,8 +64,17 @@ volumes:
     persistentVolumeClaim:
       claimName: altinn-storage-gitea-volume-claim
 
-service:
-  ports:
-    - port: 3000
-      targetPort: 3000
-      protocol: TCP
+ingress:
+  - metadata:
+      name: altinn-repositories-base-path
+      annotations:
+        # traefik.ingress.kubernetes.io/request-modifier: "ReplacePathRegex: ^/repos/(.*) /$1"
+        # traefik.ingress.kubernetes.io/request-modifier: AddPrefix: /users
+        # traefik.ingress.kubernetes.io/rewrite-target: /users
+        traefik.ingress.kubernetes.io/rule-type: PathPrefixStrip
+        traefik.frontend.priority: "500"
+        # kubernetes.io/ingress.class: nginx
+        # kubernetes.io/tls-acme: "true"
```

Comment:
The "commented out"-code should be kept for future reference or removed?

Reply:
Removed.